appium/sample-scripts/python/testdroid_ios.py

```diff
@@ -27,6 +27,9 @@ def setUp(self):
         self.screenshot_dir = os.environ.get('TESTDROID_SCREENSHOTS') or "/absolute/path/to/desired/directory"
```

Comment:
at line 26:
os.environ.get('TESTDROID_APP') or "sample/BitbarIOSSample.ipa" would work as default. Broker knows how to use this value.